### PR TITLE
Example guide CVODE update (currently without Deep Dive)

### PR DESCRIPTION
**Pull request by danmm16**
Added example documentation in RestructuredText for CVODE. Complete aside from the deep dive that was promised.

### CODE REVIEW

File: danmm.lnk
Comment (balos1) [inline danmm.lnk:1]: This file should be removed

File: doc/cvode/examples/source/_templates/layout.html
Comment (balos1) [inline doc/cvode/examples/source/_templates/layout.html:1]: The `_templates` and `_themes` directories need to be removed (they should not be committed).

**Review by balos1 — CHANGES_REQUESTED** [review]
Does this PR include documentation that refers to new examples that will come in PR #302?

File: doc/cvode/examples/source/conf.py
Comment (balos1) [inline doc/cvode/examples/source/conf.py:1]: Ah, I see this is using the old Bootstrap theme. I will update it to use the readthedocs theme. This explains why `_templates` and `_themes` were committed.

**Comment by drreynolds** [comment]
I'm closing this PR, since it includes many changes that are *not* relegated to only `doc/cvode`.  I'll reopen a different PR that includes only the relevant documentation changes.

**Pull request closed without merging**